python/src/aiconfig/Config.py

```diff
@@ -207,7 +207,8 @@ async def resolve(
 
         if prompt_name not in self.prompt_index:
             raise IndexError(
-                "Prompt not found in config, available prompts are:\n {}".format(
+                "Prompt '{}' not found in config, available prompts are:\n {}".format(
```

Comment:
nit: prefer f-strings

```suggestion
                f"Prompt '{promp_name}' not found in config, available prompts are:\n {list(self.prompt_index.keys())}"
```